chemprop/utils/v1_to_v2.py

```diff
@@ -37,8 +37,11 @@ def convert_state_dict_v1_to_v2(model_v1_dict: dict) -> dict:
         ).unsqueeze(0)
 
     for i in range(args_v1.ffn_num_layers):
-        state_dict_v2[f"predictor.ffn.{i*3}.weight"] = state_dict_v1[f"readout.{i*3+1}.weight"]
-        state_dict_v2[f"predictor.ffn.{i*3}.bias"] = state_dict_v1[f"readout.{i*3+1}.bias"]
+        suffix = 0 if i == 0 else 2
+        state_dict_v2[f"predictor.ffn.{i}.{suffix}.weight"] = state_dict_v1[
+            f"readout.{i*3+1}.weight"
+        ]
+        state_dict_v2[f"predictor.ffn.{i}.{suffix}.bias"] = state_dict_v1[f"readout.{i*3+1}.bias"]
```

Comment:
```suggestion
        LINEAR_INDEX = -1
        state_dict_v2[f"predictor.ffn.{i}.{LINEAR_INDEX}.weight"] = state_dict_v1[
            f"readout.{i*3+1}.weight"
        ]
        state_dict_v2[f"predictor.ffn.{i}.{LINEAR_INDEX}.bias"] = state_dict_v1[f"readout.{i*3+1}.bias"]
```
the linear layer is always the "last" layer in a block (even for 1-layer blocks)